entity-service/.env.example

```
@@ -1,3 +1,4 @@
+SERVER_PORT="<YOUR_SERVER_PORT>"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)